modules/environment-roles/templates/shared_iam_terraform_policy.json.tpl

```diff
@@ -25,6 +25,8 @@
         "arn:aws:iam::${account_id}:policy/TDRApiUpdatePolicy",
         "arn:aws:iam::${account_id}:policy/TDRBackendChecks*Policy${title(environment)}",
         "arn:aws:iam::${account_id}:policy/TDRMetadataChecksPolicy${title(environment)}",
+        "arn:aws:iam::${account_id}:policy/TDRMetadataChecksAPIGatewayStepFunctionExecutionPolicy${title(environment)}",
+        "arn:aws:iam::${account_id}:policy/TDRAPIInvokePolicy${title(environment)}",
```

Comment:
Can you please move this to line number 25 to maintain the order?

`"arn:aws:iam::${account_id}:policy/TDRAPIInvokePolicy${title(environment)}"`